tests/returntypes/test_jax_jit_qnode_new.py

```diff
@@ -0,0 +1,2003 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration tests for using the JAX-JIT interface with a QNode"""
+
+import pytest
+
+import pennylane as qml
+from pennylane import numpy as np
+from pennylane import qnode
+from pennylane.tape import QuantumTape
+
+qubit_device_and_diff_method = [
+    ["default.qubit", "backprop", "forward", "jax"],
```

Comment:
Do we need this first parametrization with the jax-python interface?

Reply:
This is backprop, no interface is being applied. I think it's worthwhile to have because still we have some integration tests being run for the backprop device.